docs/release-notes/2.42.md

```diff
@@ -0,0 +1,104 @@
+
+We encourage you to explore these experimental dashboards and provide feedback in the PMM forum to help us refine them.
+
+![!image](../_images/MongoDB_Cluster_Summary.png)
+
+![!image](../_images/ReplSet_Dashboard_New.png) 
+
+### Improvements
+
+- [PMM-3303](https://perconadev.atlassian.net/browse/PMM-3303) - We've introduced an experimental PMM Health dashboard in the Experimental folder, offering detailed insights into PMM's health and performance by covering key components such as node health, Query Analytics, Grafana, VictoriaMetrics, ClickHouse, and PostgreSQL metrics.
+
+- [PMM-13123](https://perconadev.atlassian.net/browse/PMM-13123) - Enhanced the **Query Response Time Distribution** graphs in the **MySQL Query Response Time Details** dashboard to provide more granular insights into query performance. Previously, the graphs did not include buckets for queries with response times less than 100 milliseconds, which is where many queries typically fall. This limitation made it difficult to analyze the performance of sub-100ms queries, especially those in the sub-1ms range.
+
+- [PMM-13075](https://perconadev.atlassian.net/browse/PMM-13075) - Added support for Ubuntu 24.04 support.
+
+- [PMM-12896](https://perconadev.atlassian.net/browse/PMM-PMM-12896), [PMM-12895](https://perconadev.atlassian.net/browse/PMM-12895), [PMM-12971](https://perconadev.atlassian.net/browse/PMM-12971) - Added per-database metrics collection and control of parameters, enabling more granular data gathering without overusing connections.
+
+- [PMM-12994](https://perconadev.atlassian.net/browse/PMM-12994) - Added labels to the `pg_replication_slot_slot_is_active` and `slot_current_wal_lsn` metrics in the postgres_exporter to identify the replication type (logical or physical) and the plugin used for the slot.
+
+- [PMM-12753](https://perconadev.atlassian.net/browse/PMM-12753) - Added experimental versions of the MongoDB Cluster Summary and ReplicaSet dashboards with a streamlined design and simplified navigation, focusing on essential metrics for monitoring MongoDB performance.
+
+- [PMM-11583](https://perconadev.atlassian.net/browse/PMM-11583) - Added support for the `innodb_redo_log_capacity` variable introduced in MySQL 8.0, ensuring accurate and consistent data representation in the InnoDB Logging graphs.
+
+### Fixed issues
+
+- [PMM-12349](https://perconadev.atlassian.net/browse/PMM-12349) - Fixed an issue in the ReplicaSet Summary dashboard where incorrect data was displayed when a node in the replica set was down, ensuring that the dashboard accurately reflects the status and version information of the nodes even when they are unavailable.
+
+- [PMM-12522](https://perconadev.atlassian.net/browse/PMM-12522) - Fixed a limitation in PMM Server that prevented it from monitoring more than 8 mongos instances in a large MongoDB cluster deployment.
```

Comment:
I think this should be re-phrased because the real problem wasn't related to the amount of mongos instances